### ISSUE TITLE
Cant find Dyson local and Dyson Cloud in Integration

### ISSUE THREAD

**Issue by zsw12abc** [shenxn/ha-dyson#158]
Hey mate,
i have install both Dyson Cloud and Dyson Local in my Home Assistant
and restart it.
after that, i still cant find anything via searching Dyson in my Configuration -> Integrations
![image](https://user-images.githubusercontent.com/11704698/184806326-c8358bc2-76b9-45a7-9ac1-680258da3776.png)
Anything i missed?

**Comment by zsw12abc** [comment]
sorry mate,
I can find those now. But when i try to add them, error messages pops up

it says that this is not support to config via UI 

![image](https://user-images.githubusercontent.com/11704698/184830751-0074490c-7cbe-4d68-a29f-aebceceb1c75.png)

**Comment by cachrisman** [comment]
yeah I get the same for both Dyson Cloud and Dyson Local. I've installed HACS and added the custom repository and downloaded both the dyson cloud and dyson local custom integration. They now show up in my Add Integrations list:

![image](https://user-images.githubusercontent.com/6521666/185584980-45aa8069-4193-4e30-8353-3e43573609b1.png)

but when I try to set either up I get the following error:

> Error
> 
> This integration does not support configuration via the UI. If you followed this link from the Home Assistant website, make sure you run the latest version of Home Assistant.
> 
> 

![image](https://user-images.githubusercontent.com/6521666/185584532-9cde67c6-7649-47ba-971b-a347a90064d2.png)

I'm running Home Assistant Core 2022.8.5 on HAOS 8.2.

**Comment by zsw12abc** [comment]
now i can see the pop up but still having the error while log in via Dyson Cloud or enter the SSID and WIFI password via Dyson Local 
![image](https://user-images.githubusercontent.com/11704698/185780617-8cb562e3-a425-4da9-ae2c-ef2f583db9c7.png)

**Comment by snigehere** [comment]
I got this error because I was still logged onto the dyson cloud via my app.  I logged off the app and was then able to login via HA

**Comment by zsw12abc** [comment]
> I got this error because I was still logged onto the dyson cloud via my app. I logged off the app and was then able to login via HA

Hey mate,
does that mean only one of HA and App can connect via cloud?

**Comment by snigehere** [comment]
There was a discussion in that but I don't think there was a conclusion.  My own opinion is yes, only one cloud connection. I used the HA cloud integration just to get the settings for the local integration.  I then removed the HA cloud as I didn't need both so now have HA local and my phone app connected to the cloud.

**Comment by zsw12abc** [comment]
> There was a discussion in that but I don't think there was a conclusion. My own opinion is yes, only one cloud connection. I used the HA cloud integration just to get the settings for the local integration. I then removed the HA cloud as I didn't need both so now have HA local and my phone app connected to the cloud.

When you mention HA Local, are u talking about the Dyson Local?  but i tried to use dyson Local to connect it but still failed.

**Comment by snigehere** [comment]
Yes - I mean HA dyson local ... Initially I had issues with installing the two integrations but a fix was made and I was then able to proceed.  

I use HACS to download both the dyson cloud then dyson local packages.  I then used the documentation to do the following:

step 1 was to add the HA Dyson cloud integration (logging out of the dyson app on my phone ahead of doing this)
step 2 was to configure and start the HA dyson cloud integration - this provided the data to be able to do step 3
step 3 was to configure and start the HA Dyson local integration
step 4 was to remove the HA dyson cloud integration (I still have it in HACS but the integration has been removed from settings>integrations)
step 5 was to log back onto my dyson app on the phone.

I can now use HA to see, via the dsyon local integration, the info from the device and also control the device.

**Comment by haddood** [comment]
> yeah I get the same for both Dyson Cloud and Dyson Local. I've installed HACS and added the custom repository and downloaded both the dyson cloud and dyson local custom integration. They now show up in my Add Integrations list:
> 
> ![image](https://user-images.githubusercontent.com/6521666/185584980-45aa8069-4193-4e30-8353-3e43573609b1.png)
> 
> but when I try to set either up I get the following error:
> 
> > Error
> > This integration does not support configuration via the UI. If you followed this link from the Home Assistant website, make sure you run the latest version of Home Assistant.
> 
> ![image](https://user-images.githubusercontent.com/6521666/185584532-9cde67c6-7649-47ba-971b-a347a90064d2.png)
> 
> I'm running Home Assistant Core 2022.8.5 on HAOS 8.2.

same issue here, anybody figured away to make it work?

**Comment by LunchboxNYC** [comment]
> Yes - I mean HA dyson local ... Initially I had issues with installing the two integrations but a fix was made and I was then able to proceed.
> 
> I use HACS to download both the dyson cloud then dyson local packages. I then used the documentation to do the following:
> 
> step 1 was to add the HA Dyson cloud integration (logging out of the dyson app on my phone ahead of doing this) step 2 was to configure and start the HA dyson cloud integration - this provided the data to be able to do step 3 step 3 was to configure and start the HA Dyson local integration step 4 was to remove the HA dyson cloud integration (I still have it in HACS but the integration has been removed from settings>integrations) step 5 was to log back onto my dyson app on the phone.
> 
> I can now use HA to see, via the dsyon local integration, the info from the device and also control the device.

How did you get you configure the HA Dyson Cloud integration, did you use the UI or the config file?  If you use date config can you post an example?
